src/components/HeadOfTable.tsx

```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export const HeadOfTable: React.FC = () => {
+  const handleNavLinkClass = ({ isActive }: { isActive: boolean }) => {
+    return classNames('navbar-item', {
+      'has-background-grey-lighter': isActive,
+    });
+  };
```

Comment:
It is better to define this function outside the component so that it is not created every time a new render is performed.